ClingoFluent command
ClingoFluent sense_fluent
---
bool success
string status
ClingoFluent[] observable_fluents
---
string status
ClingoFluent[] observable_fluents
